core/src/main/java/pl/netigen/core/utils/Utils.java

```diff
@@ -42,7 +42,7 @@ public static boolean isPermissionGranted(AppCompatActivity activity, String per
 
     @NonNull
     private static String getMarketLink(Activity activity, String packageName) {
-        int link_id = Config.Companion.isSamsung() ? R.string.samsung_app_id_link_netigen : R.string.play_app_id_link_netigen;
+        int link_id = Config.Companion.isSamsungStatic() ? R.string.samsung_app_id_link_netigen : R.string.play_app_id_link_netigen;
```

Comment:
Why change good old name isSamsung()?

Reply:
Changed it already in https://github.com/netigenkluzowicz/api_android/pull/74/commits/bd04cb75bb348e26b3e15c039dfb254f17da632f